logbook-json/src/main/java/org/zalando/logbook/json/StaticParameterReplacementOperator.java

```diff
@@ -0,0 +1,21 @@
+package org.zalando.logbook.json;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+
+import java.util.function.BinaryOperator;
+
+import static org.zalando.logbook.json.PrimitiveJsonPropertyBodyFilter.quote;
+
+@AllArgsConstructor
+@EqualsAndHashCode
+public class StaticParameterReplacementOperator<T> implements BinaryOperator<String> {
```

Comment:
```suggestion
final class StaticParameterReplacementOperator<T> implements BinaryOperator<String> {
```

Always try to limit the surface of your API to the absolute minimum. Gives you freedom to refactor and change stuff later on.